website/templates/share_registration.mako

```diff
@@ -0,0 +1,19 @@
+<%inherit file="base.mako"/>
+<%def name="title()">SHARE</%def>
+<%def name="content()">
+    <div id="share_registration_iframe"></div>
+</%def>
+
+<%def name="javascript_bottom()">
+    <script>
+        // Mako variables accessible globally
+        window.contextVars = $.extend(true, {}, window.contextVars, {
+            share: {
+                urls: {
+                    registrationURL: '${registrationURL | js_str}'
```

Comment:
`URL` is redundant and defeats the purpose of the `urls` namespace`; `share -> urls -> register` is more concise.